tests/unit/s2n_security_policies_test.c

```diff
@@ -168,39 +148,80 @@ int main(int argc, char **argv)
             EXPECT_TRUE(has_tls_13_sig_alg);
             EXPECT_TRUE(has_rsa_pss);
         }
+
+        /* TLS 1.3 + PQ checks */
+        if (security_policy->kem_preferences->tls13_kem_group_count > 0) {
+            /* Ensure that PQ enabled Policies support TLS 1.3 since TLS 1.3 is now required for PQ support. */
+            EXPECT_TRUE(has_tls_13_cipher);
```

Comment:
Moved "TLS 1.3 + PQ checks" further down in the function so that this code has access to the `has_tls_13_cipher` variable. This move allows adding this check that requires TLS 1.3 support for all PQ policies.